definitions/npm/commander_v2.x.x/flow_v0.28.x-/commander_v2.x.x.js

```diff
@@ -159,7 +159,12 @@ declare module "commander" {
      * @return {Command} for chaining
      * @api public
      */
-    parse(argv: Array<string>): this;
+    parse(argv: Array<string>): {
+      ...this,
+      args: Array<string>,
+      rawArgs: Array<string>,
+      [string]: any,
+    };
```

Comment:
I am not sure that this is the best way to fix it.
If you have a good idea to modify this, tell me please.

Reply:
Other way could be `this & { args: Array<string>, ... }`. I'm not sure which is better.

Reply:
I like your answer. `this & { args: Array<string>, ... }` make this more understandable.
I will change.